std_msgs/Bool command
---
std_msgs/String name
